src/ios/CDVLocation.h

```diff
@@ -47,10 +47,12 @@ typedef NSUInteger CDVLocationStatus;
     @private BOOL __locationStarted;
     @private BOOL __highAccuracyEnabled;
     CDVLocationData* locationData;
+    dispatch_queue_t queue;
```

Comment:
If `queue` is only used on a single method, why is it a class property?

Reply:
The fund is called multiple times, so every time it's called it's added to the queue, the alternative is to create a queue every time the func is called.

Reply:
Still, only `isLocationServicesEnabledWithCompletion` deals with it so, in terms of encapsulation, wouldn't it make sense for it to be a local variable?